backend/src/config/passport.ts

```diff
@@ -29,7 +29,7 @@ const authStrategy = (passport: passport.PassportStatic): void => {
 		"local-signup",
 		new LocalStrategy(
 			{
-				usernameField: "username",
+				usernameField: "email",
```

Comment:
Email makes more sense as unique identifier. This changes the key passport uses as the identifier